Apps/Contoso.UWP.Puppet/MainPage.xaml

```diff
@@ -13,5 +13,12 @@
             <Button Content="Throw exception" Click="ThrowException" />
             <ToggleSwitch Header="Handle all exceptions" x:Name="HandleExceptions" Margin="20,0,0,0" />
         </StackPanel>
+        <StackPanel Orientation="Vertical">
```

Comment:
This is fine but CTI does not test this app.

We should rather modify the Xamarin.Forms app instead (Contoso.Forms.Puppet) since the warning is available for Xamarin too.